src/test/scala/nl.knaw.dans.easy.deposit/DepositDirSpec.scala

```diff
@@ -180,6 +180,6 @@ class DepositDirSpec extends TestSupportFixture with MockFactory {
   }
 
   private def createDepositAsPreparation(user: String) = {
-    DepositDir.create(draftsDir, user).getOrRecover(e => fail(e))
+    DepositDir.create(draftsDir, user).getOrRecover(e => fail(e.toString))
```

Comment:
let's keep those stack traces in there. `fail(e.toString, e)`
better for fixing failing tests, IMO.

same for the others